packages/forklift-console-plugin/src/modules/Plans/views/details/components/UpdateMappings/PlanMappingsInitSection.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
Originally from [here](https://github.com/kubev2v/forklift-console-plugin/blob/b3f66068b55b9985f98edd50a51e4f5400a5dc91/packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/Mappings/PlanMappings.tsx#L53)

Reply:
Lifted up since it's used in multiple locations now